### PR TITLE
Add various comments in the codebase

### PR DESCRIPTION
**Pull request by 94noni**
Add some comments while reading some source code
Ref https://symfony.com/doc/current/setup.html#the-symfony-demo-application

### CODE REVIEW

File: src/Controller/Admin/BlogController.php [hunk 1]
```diff
@@ -30,6 +30,7 @@
  * purposes. However, in your real Symfony application you should use any of the
  * existing bundles that let you generate ready-to-use backends without effort.
  *
+ * See https://symfony.com/bundles/EasyAdminBundle/current/index.html
  * See http://knpbundles.com/keyword/admin
```
Comment (stof) [inline src/Controller/Admin/BlogController.php:34]: I suggest removing that knpbundles.com link as knpbundles.com does not seem functional anymore (it has 0 bundles referenced in it)

File: src/Controller/BlogController.php [hunk 2]
```diff
@@ -75,12 +75,17 @@ public function index(Request $request, int $page, string $_format, PostReposito
     #[Route('/posts/{slug}', methods: ['GET'], name: 'blog_post')]
     public function postShow(Post $post): Response
     {
-        // Symfony's 'dump()' function is an improved version of PHP's 'var_dump()' but
+        // Symfony's 'dump()' and 'dd()' functions are an improved version of PHP's 'var_dump()' but
```
Comment (stof) [inline src/Controller/BlogController.php:78]: ``dd` is not just an improved `var_dump`. It is dump + die, which is quite different that just dumping.

Comment (94noni) [reply src/Controller/BlogController.php:78]: you are right, now the phrasing is weird
do you think I revert and adding the `dd()` is not worth it? or keep it but rephrase?
thank you @stof

**Comment by javiereguiluz** [comment]
Thanks Antoine!

File: src/Controller/Admin/BlogController.php [hunk 3]
```diff
@@ -30,7 +30,7 @@
  * purposes. However, in your real Symfony application you should use any of the
  * existing bundles that let you generate ready-to-use backends without effort.
  *
- * See http://knpbundles.com/keyword/admin
+ * See https://symfony.com/bundles/EasyAdminBundle/current/index.html
```
Comment (javiereguiluz) [inline src/Controller/Admin/BlogController.php:33]: I've replaced this link by `https://symfony.com/bundle` There are several admin bundles, so it's not fair to link specifically to one of them 👍

**Pull request merged**